File: 2D_amber/Assets/Scripts/CSharp/Map/CoinObject.cs
using UnityEngine;
using System.Collections;

public class CoinObject : MonoBehaviour {
	public int amount = 1 ;
	
	void OnTriggerEnter2D(Collider2D co){
		if(co.tag != "Player"){return;}
		Game.sav.GainMoney(1) ;
		GetComponent<Animator>().SetTrigger("Eat") ;
		Destroy(this) ;
	}
}

File: 2D_amber/Assets/Scripts/JS/Map/CoinObject.js
#pragma strict
var amount : int = 1 ;

function OnTriggerEnter2D(co:Collider2D){
  if(co.tag != "Player"){return;}
  Game.sav.GainMoney(1) ;
  GetComponent.<Animator>().SetTrigger("Eat") ;
  Destroy(this) ;
}

